src/MergeContext.js

```diff
@@ -787,9 +787,21 @@ class PullRequest {
         return (this._rawPr.title + ' (#' + this._rawPr.number + ')' + '\n\n' + this._prBody()).trim();
     }
 
+    _invalidCharacterPosition(str) {
```

Comment:
Please document whether this function returns the number of octets/bytes or the number of utf-8 characters leading to the first invalid character.